packages/verification-vue-component/src/components/Verification/VerificationDropBox.vue

```diff
@@ -48,6 +63,21 @@ export default {
         },
         filesDropped(target, files) {
             this.$emit('files-dropped', files)
+        },
+        dragOver() {
+            if (!this.draggingOver) {
+                this.draggingOver = true
+                this.dragLeaveLocked = true
+
+                setTimeout(() => {
```

Comment:
It came just to my mind, maybe you could also try to use `await this.$nextTick()` instead of timeout. This would wait for the next DOM update cycle, this could already be enough. You could just add it like this:
```
await this.$nextTick()
this.dragLeaveLocked = false
```